From 3f953b8ef848cacbdf9aabee6b3562ebc63b7ea1 Mon Sep 17 00:00:00 2001
From: Damien Thenot <damien.thenot@vates.tech>
Date: Mon, 11 Aug 2025 14:04:24 +0200
Subject: [PATCH] fix(qcow2util): read clusters with kill data

Reading allocation table is needed for kill data to run

Signed-off-by: Damien Thenot <damien.thenot@vates.tech>
---
 drivers/qcow2util.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/qcow2util.py b/drivers/qcow2util.py
index 8a745b30..a929e2e2 100644
--- a/drivers/qcow2util.py
+++ b/drivers/qcow2util.py
@@ -680,7 +680,7 @@ class QCowUtil(CowUtil):
         Returns:
             nothing.
         """
-        self._read_qcow2(path)
+        self._read_qcow2(path, read_clusters=True)
         # We need to reset L1 entries and then just truncate the file right
         # after L1 entries
         with open(self.filename, "r+b") as file:
